src/gdrive.jl

```diff
@@ -30,26 +34,28 @@ function gsheet_handler(url; format=:csv)
     return url
 end
 
-function gfile_handler(url)
+function prepare_gfile_url(url)
     # pattern of file path in google drive:
     # https://drive.google.com/file/d/<hash>/view?usp=sharing
     h = match(r"/file/d/([^\/]+)/", url)
-    (h === nothing) && throw("Can't find goole drive file ID in the url")
-
-    return "https://docs.google.com/uc?export=download&id=$(h.captures[])"
+    if isnothing(h)
+        throw(ErrorException("Can't find goole drive file ID in the url"))
+    else
+        return "https://docs.google.com/uc?export=download&id=$(h.captures[])"
+    end
```

Comment:
How about 

```julia
isnothing(h) && throw(ErrorException("Can't find google drive file ID in the url"))

return "https://docs.google.com/uc?export=download&id=$(h.captures[])"
```

So as to match the style

Reply:
To clarify the semantics, `if...else` clause expresses better.

Reply:
The way I mentioned match the `Return Early Pattern` that make the return more explicitly for the function.